src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/10.md

```diff
@@ -36,14 +36,14 @@ In 1751, painter William Hogarth created a pair of prints named ‘Beer Street
+It sounds unlikely, weird even, but the first truly global beer style (e.g., porter) had eventually all but vanished and was not produced at all in the United Kingdom from the beginning of the 1950s until 1978. Stouts were luckier, as they were (and are) brewed by plenty of companies, starting with an international giant, Guinness. Nevertheless, one should take into account that contemporary porters and stouts are brewed using more modern technology, including with those of the second half of the 19th century (see the ‘Age of Empires’ chapter), and were wholly distinct from the original ‘entire-butt’ porters. Several reconstructions exist, namely ‘Entire Butt English Porter’ by Salopian Brewery, but it's rather hard to find them. The best approximation of the early English porter is actually the brown ale of Flanders, the story of which we will tell in Section II of this book.
 
 If we talk about modern porters, the most hailed ones are English Fuller's London Porter and Samuel Smith Taddy Porter and Scottish Harviestoun Old Engine Oil.
 
-#### The trust that went bust
+#### The Trust that went Bust
 
-The Hogarth's diptych graphically outlines the processes that were brewing in 18th-century England. Being one the most potent locomotives of the Industrial revolution and accumulation of capital, beer production was finally pushed back by more advanced technologies. The tea, coffee, soft drinks, and strong alcohol industries were blooming and had soon become competitive powers (partly because of the re-investment of capital made on brewing). Wealthy social stratum still preferred prestigious grape wine over beer; less prosperous common folk had switched to cheap gin[ref:change-is-brewing:252-253]().
+Hogarth's diptych graphically outlines the brewing processes in 18th-century England. Being one the first potent locomotives of the Industrial revolution and accumulation of capital, beer production was ultimately pushed aside by more advanced technologies. The tea, coffee, soft drinks, and strong alcohol industries were blooming and had soon become competitive powers (partly because of the re-investment of capital originally earmarked for brewing). Wealthy social stratum still preferred prestigious continental wine over beer; less prosperous common folk had switched to cheap gin[ref:change-is-brewing:252-253]().
```

Comment:
The intent is rather: they made mpney on beer, but invested it in the beer substitutes. So not 'earmarked', but 'made on'.